### DISCUSSION TITLE
Email SMTP Notification setup

### DISCUSSION THREAD

**Discussion by Bammede** [upptime/upptime#314] [Q&A] [answered]
https://upptime.js.org/docs/notifications#smtp

How to setup Email SMTP notification? I am newbie and set my upptime with a step-by-step guide but cannot find manual to set email smtp notification. I mean, should I put this code in upptimerc.yml like this:

```
NOTIFICATION_EMAIL_SMTP=true
NOTIFICATION_EMAIL_SMTP_PORT=587
NOTIFICATION_EMAIL_SMTP_HOST=hostname
NOTIFICATION_EMAIL_SMTP_USERNAME=username
NOTIFICATION_EMAIL_SMTP_PASSWORD=password
```

**Comment by AnandChowdhary** [comment, marked as answer]
No, you should add these as repository secrets (environment variables): https://upptime.js.org/docs/notifications

**Reply by Bammede** [reply]
Is that simple? Thank you!
